doc/requirements.rst

```diff
@@ -130,6 +136,8 @@ Motor versions.
 +---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+
 |               | 2.0 |**N**|**N**|  Y  |**N**|  Y  |**N**  |  Y    |  Y  |  Y  |
 +---------------+-----+-----+-----+-----+-----+-----+-------+-------+-----+-----+
+|               | 2.1 |**N**|**N**|  Y  |**N**|  Y  |**N**  |  Y    |  Y  |  Y  |
```

Comment:
@ajdavis why do we break out 3.5 into 3.5.0 and 3.5.2? We don't specify micro version numbers for any other Python runtime version... Should I consolidate this into a single 3.5 column?

Reply:
https://github.com/mongodb/motor/commit/d49ff6551a95c8b5dc0bcae1096dda8010e80492#diff-f53de7fd54b727d055c43104e20d05f7

Reply:
See the notes above the table.

Reply:
Ah, my bad. Thanks for pointing it out.